data/benefits/javascript/martinique-aide-territoriale-etudiants.yml

```diff
@@ -22,14 +21,14 @@ conditions_generales:
       - "02"
   - type: quotient_familial
     operator: "<="
-    value: 28000
+    value: 28500
```

Comment:
C'est un peu perturbant car dans le texte, j'ai trouvé : 
`;
Ne pas dépasser la limite du quotient familial fixé par la CTM à savoir 28 000 € (Avis imposition de l’année N) ;`

Mais il y a aussi : 
    De 1 400 € si QF< 18 190€
    De 1 100 € si 18 190 € <QF< 22 500 €
    De 9 00 € si 22 500 € <QF< 28 500 €

Cela pourrait même être une aide OF mais pas évidente à faire.

Reply:
Je pense que dans le doute, l'idéal est effectivement d'harmoniser les QF sur toutes les aides de la Martinique pour les étudiants, à 28 000 € donc.